syntax = "proto3";

package todo;

service TodoService {
  rpc AddTask(TaskRequest) returns (TaskResponse);
  rpc GetTasks(Empty) returns (TasksList);
}

message TaskRequest {
  string task = 1;
}

message TaskResponse {
  string message = 1;
}

message TasksList {
  repeated string tasks = 1;
}

message Empty {}
